protool/__init__.py

```diff
@@ -13,6 +13,7 @@
 import sys
 import tempfile
 from typing import Any, Dict, List, Optional
+from OpenSSL.crypto import FILETYPE_ASN1, load_certificate, x509
```

Comment:
I'd prefer if these weren't imported directly in. i.e. do
```
import OpenSSL
```

Then use as:

```
_ = OpenSSL.crypto.load_certificate(...)
```

It makes it clearer about where things come from.

Reply:
It also solves a re-declaration problem below.

Reply:
These should be alphabetical by package name. i.e. this entry comes just before `import os`